### ISSUE TITLE
tasks.teachosm.org move

### ISSUE THREAD

**Issue by dalekunce** [hotosm/tech#102]
The Red Cross has hosted tasks.teachosm.org for the past few years. The deployed version is TM2 on a t2.micro instance. Because of some internal IT rules we can no longer host the server. We need to find a new home and hopefully upgrade it to TM3.

cc @smit1678

**Comment by geomantic** [comment]
@dalekunce We're moving forward with plans to migrate and been in touch with @smit1678 to explore options.

**Comment by geomantic** [comment]
@smit1678 Hooray! I believe @mikelmaron owns the TeachOSM domain.

**Comment by dalekunce** [comment]
great to see this moving forward let me know if I can help in any way.

**Comment by shawnmgoulet** [comment]
Looks like one can no longer use the "login to OpenStreetMap" button on the TOSM TM. It's directing to "An error occurred" from ngix.

Also, the TOSM TM site doesn't recognize I've signed into OSM if I do that 1st, then refresh the TM site.

Is this associated with the ongoing move or unrelated? @smit1678 @dakotabenjamin

**Comment by dakotabenjamin** [comment]
I've got a working instance (upgraded to TM3!) at http://tasks-teachosm.hotosm.org/

Please look around and see if it's working to your liking. When we are ready to switch over, we will have to change the domain to tasks.teachosm.org

**Comment by shawnmgoulet** [comment]
Awesome @dakotabenjamin!

Looks to be working well to me.

**Comment by geomantic** [comment]
@dakotabenjamin Looks to be working well. Just a configuration detail: Can the default tag be changed from #hotosm-project-xxx  to #teachosm-project-xxx ?

**Comment by dakotabenjamin** [comment]
@geomantic done. It looks like @mikelmaron has control over the domain- you can DM me if you're on slack or shoot me an email dakota.benjamin at hotosm

**Comment by geomantic** [comment]
@dakotabenjamin @mikelmaron Were you guys able to transfer the TeachOSM domain registration?

**Comment by dakotabenjamin** [comment]
OK I've created a branch in https://github.com/hotosm/tasking-manager/ called `teachosm` that is set up with continuous integration, so any changes can be pushed there and they will be updated automatically.

**Comment by shawnmgoulet** [comment]
Sweet Dakota. Will that code base be exposed through [tasks.teachosm.org](https://tasks.teachosm.org/)?

**Comment by dakotabenjamin** [comment]
Once the domain name transfer is completed, yes.

**Comment by geomantic** [comment]
Many thanks @dakotabenjamin A couple questions:
1 were you able to get in touch w/ Ian Dees/OSMUS to have the TeachOSM domain transferred? 
2 can we incorporate the edits to the landing page (color, text) before going live? I believe @smit1678 has them per Nuala Cowan.

**Comment by dakotabenjamin** [comment]
Yeah, Mikel and Ian are currently handling the transfer. We can hold off pointing it to the new app until you all are comfortable with the look of it.

Whatever the changes, make a pull request to the `teachosm` branch. They will be deployed automatically. I appreciate feedback on how that deployment process works. We can work on creating a system for monitoring and approving those changes as well.

**Comment by geomantic** [comment]
@dakotabenjamin okay, sounds good. also, pinged you on slack in re: the OAuth key. Too many platforms... 

@shawnmgoulet ^^

**Comment by geomantic** [comment]
@dakotabenjamin @smit1678 Hey guys, just checking in on progress toward completing the upgrade. Requests to use the TM are piling up and we'd like to be able to offer users a date when it will be completed. Any updates? Thanks!

**Comment by smit1678** [comment]
@geomantic Last I understand is that the domain is being transferred and still in flux. I don't think this should hold up pointing the domain at instance. @mikelmaron do you still have access to this? can you create an ALIAS record and point it at `taskingmanager-teachosm.us-east-1.elasticbeanstalk.com`?

The temp domain is working and so if you want to point people here: https://tasks-teachosm.hotosm.org/. We'll then make sure that the domain is forwarded to tasks.teachosm.org once it is hooked up.

**Comment by dakotabenjamin** [comment]
Is there someone tasked with making the proposed style changes? I think that's the only holdup. If no one, I can do it by next Tuesday but I am currently traveling. Then Ian/OSM needs to change the record set on route53 to the elastic beanstalk domain.

**Comment by smit1678** [comment]
I started poking around with some ways we can adjust, I'll try to get that on the branch later this week.
